tests/integration/mutation/delete/simple_test.go

```diff
@@ -55,7 +55,7 @@ func TestMutationDeletion_WithoutSubSelectionFields(t *testing.T) {
 
 					}
 				}`,
-				ExpectedError: "Syntax Error GraphQL request (2:17) Unexpected empty IN {}",
```

Comment:
Question: What happened here?

Reply:
It was behaving weird, if I left the `request (2:17)` in it would fail on some tests matrix but pass others. Basically, some of the tests were including the `request` substring, and others weren't. I dont know why its acting up, but it doesn't semantically change anything. The `(2:17) Unexpected empty IN {}` is still there though.